healthcheck/healthcheck.py

```diff
@@ -116,18 +117,30 @@ class FilesExistHealthCheck(ListHealthCheck):
     """Fails if at least one of passed files doesn't exist."""
 
     def check_item(self, filename):
-        file_exists = os.path.isfile(filename)
-        details = {filename: 'exists' if file_exists else 'NO SUCH FILE'}
-        return file_exists, details
+        try:
+            ok = file_exists(filename)
+        except OSError as e:
+            ok = False
+            description = 'ERROR: ' + str(e.strerror)
```

Comment:
`None` sounds reasonable. I think PyCharm is wrong about it being an int, though.